doc/contribute/project_roles.rst

```diff
@@ -0,0 +1,283 @@
+:orphan:
```

Comment:
why orphan? I would make sure this is linked in the table of contents somewhere instead of just via a reference.

Reply:
It is linked in the index.rst of contribute/ section

Reply:
now present also in  the table of contents